assets/emoji-seq.yaml

```
@@ -9,3 +9,6 @@
     - <:hara:991347806180741160>
     - <:ki:991347820693032970>
     - <:ri:991347831875047454>
+- pattern: 磯崎
```

これは正規表現のパターンを書くようになっているので, こうしておいたほうが便利かも
```suggestion
- pattern: 磯(崎|﨑)
```